src/component/page/bruker-detaljer/deltaker-detaljer/endre-deltaker-modal/EndreProsentDeltakelseModal.tsx

```diff
@@ -83,8 +83,16 @@ export const EndreProsentDeltakelseModal = ({
 
   const sendForslag = () => {
     const prosentDeltakelse = parseInt(prosentDeltakelseFelt)
-    const dagerPerUke =
-      prosentDeltakelse === 100 ? undefined : parseInt(dagerPerUkeFelt)
+    const dagerPerUke = prosentDeltakelse === 100
+      ? undefined
+      : (dagerPerUkeFelt.length > 0 ? parseInt(dagerPerUkeFelt) : undefined)
+
+    const harIngenEndring = prosentDeltakelse === gammelProsentDeltakelse &&
+      dagerPerUke == gammelDagerPerUke
```

Comment:
Jeg tror ikke dette fungerer helt, `gammelDagerPerUker` er `number | null` mens man her setter `dagerPerUke` til `undefined` hvis man har 100%, så jeg får lov til å foreslå 100% selv om deltakeren allerede har samme prosent.

Reply:
ah fikk du testet det? for jeg satt == og ikke ===, slik at undefined == null er true